index.js

```diff
@@ -10,68 +11,73 @@ const bookList = document.querySelector('.books');
+class Storage {
+  static getBook() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
   }
 
-  return books;
-}
-function addBook(newBook) {
-  const books = getBook();
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
-}
-function removeBook(title) {
-  const books = getBook();
-  const index = books.findIndex((book) => book.title === title);
-  if (index !== -1) {
-    books.splice(index, 1);
+  static addBook(newBook) {
+    const books = Storage.getBook();
+    books.push(newBook);
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static removeBook(title) {
+    const books = Storage.getBook();
+    const index = books.findIndex((book) => book.title === title);
+    if (index !== -1) {
+      books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(books));
+    }
+  }
 }
 
 // Populate Books from Local Storage
 
-const books = getBook();
+const books = Storage.getBook();
 books.forEach((book) => bookList.insertAdjacentHTML(
   'afterbegin',
   `<div><p>${book.title}</p> <p>${book.author}</p> <button class="remove-btn">remove</button></div>`,
 ));
 
 // Display new Books
+class UI {
+  static displayNew() {
+    const author = document.getElementById('author').value;
+    const title = document.getElementById('title').value;
+    const newBook = new Book(title, author);
 
-function displayNew() {
-  const author = document.getElementById('author').value;
-  const title = document.getElementById('title').value;
-  const newBook = new Book(title, author);
-
-  if (title && author) {
-    bookList.insertAdjacentHTML(
-      'afterbegin',
-      `<div><p>${newBook.title}</p> <p>${newBook.author}</p> <button class="remove-btn">remove</button></div>`,
-    );
-    addBook(newBook);
-    document.getElementById('author').value = '';
-    document.getElementById('title').value = '';
-  } else {
-    buttonAdd.insertAdjacentHTML(
-      'afterend',
-      '<p class="error-msg">Please enter a valid title and author<p>',
-    );
-    setTimeout(() => {
-      const errorMsg = document.querySelector('.error-msg');
-      errorMsg.remove();
-    }, 3000);
+    if (title && author) {
+      bookList.insertAdjacentHTML(
+        'afterbegin',
+        `<div><p>${newBook.title}</p> <p>${newBook.author}</p> <button class="remove-btn">remove</button></div>`,
+      );
+      Storage.addBook(newBook);
+      document.getElementById('author').value = '';
+      document.getElementById('title').value = '';
+    } else {
+      buttonAdd.insertAdjacentHTML(
+        'afterend',
+        '<p class="error-msg">Please enter a valid title and author<p>',
+      );
+      setTimeout(() => {
+        const errorMsg = document.querySelector('.error-msg');
+        errorMsg.remove();
+      }, 3000);
+    }
   }
 }
```

Comment:
- [ ] Good job so far :clap: , However, Please ensure that your UI matches the given [design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) as part of the project requirement. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The screenshot below illustrate the differences. :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student Design](https://github.com/mj-e-boc/Awesome-Books/assets/43528594/b95ee999-ca44-4533-8ef6-f539704c341b)

</td>
      <td> 

 
<img width="437" alt="awsome" src="https://github.com/mj-e-boc/Awesome-Books/assets/43528594/4288e1d3-c935-432b-95ea-9d8fe0cc9676">



</td>
    </tr>
  </table>